doc/classes/EditorExportPlugin.xml

```diff
@@ -81,8 +84,9 @@
 			<param index="1" name="type" type="String" />
 			<param index="2" name="features" type="PackedStringArray" />
 			<description>
-				Virtual method to be overridden by the user. Called for each exported file, providing arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
+				Virtual method to be overridden by the user. Called for each exported file, except for imported resources (resources that have an [code].import[/code] file). The arguments that can be used to identify the file. [param path] is the path of the file, [param type] is the [Resource] represented by the file (e.g. [PackedScene]) and [param features] is the list of features for the export.
```

Comment:
"The arguments that can be used to identify the file."
Looks like a typo.

Reply:
Ah, thanks! "associated" is a good improvement -- that part was itching the back of my mind.

And yes, that's definitely a typo that I made when adapting the existing text. Fix coming in a moment...